apps/self-hosted/DEPLOYMENT.md

```
@@ -0,0 +1,472 @@
+
+```
+To: (TBD - payment account not yet active)
+Amount: 1.000 HBD
+Memo: blog:YOUR_HIVE_USERNAME
+```
+
+---
+
+## Support
+
+- GitHub Issues: https://github.com/ecency/vision-next/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)